build/BUILD

```diff
@@ -92,7 +92,7 @@ grep ^STABLE_BUILD_SCM_REVISION bazel-out/stable-status.txt \
 genrule(
     name = "cni_package_version",
     outs = ["cni_version"],
-    cmd = "echo 0.5.1 >$@",
+    cmd = "echo 0.6.0 >$@",
```

Comment:
This means we have multiple locations for CNI version.  

Can we bake it into a single variable?

Reply:
Per our conversation. This is fine to merge as it stands @timothysc but we need to capture the requirement that the dependent version and the built version are locked to the same var.

Reply:
yes, this is what we do with the CRI version:
https://github.com/kubernetes/kubernetes/blob/8c0542dcf126222481390cd0606e3b7f43830a64/build/workspace.bzl#L15

we should do something similar here.

Reply:
though then also we should probably use that variable in the `WORKSPACE`:

https://github.com/kubernetes/kubernetes/blob/8c0542dcf126222481390cd0606e3b7f43830a64/build/root/WORKSPACE#L57-L62